syncer/sharding_group.go

```diff
@@ -379,7 +380,28 @@ func (sg *ShardingGroup) FlushData(targetTableID string) ([]string, [][]interfac
 }
 
 // GenTableID generates table ID
-func GenTableID(schema, table string) (ID string, isSchemaOnly bool) {
+func GenTableID(schema, table string, upperSchema, upperTable []string) (ID string, isSchemaOnly bool) {
+	for _, s := range upperSchema {
+		if s == schema {
```

Comment:
Maybe map will be better

Reply:
for a small count, iter through array is faster than map (and i'm lazy)